cob_moveit_config/raw3-1/config/ompl_planning.yaml

```diff
@@ -1,75 +1,68 @@
 planner_configs:
```

Comment:
Well this was updated and tested from the vision fair. So it should still be updated if this configuration was not lost with all these last changes in the repository.